tpm2_pytss/types.py

```diff
@@ -832,16 +839,16 @@ def from_pem(
             p.parameters.eccDetail.kdf.scheme = TPM2_ALG.NULL
         return p
 
-    def to_pem(self, encoding="pem"):
-        """Encode the public key in standard format.
+    def to_pem(self, public: "TPMT_PUBLIC", password: bytes = None):
+        """Encode the key as PEM encoded ASN.1.
 
-        Args:
-            encoding (str, optional): The encoding format, one of "pem", "der" or "ssh".
+        public(TPMT_PUBLIC): The corresponding public key.
+        password(bytes): An optional password for encrypting the PEM with.
 
         Returns:
-            Returns the encoded key as bytes.
+            Returns the PEM encoding as bytes.
         """
-        return _public_to_pem(self, encoding)
+        return _public_to_pem(self, public, password)
 
```

Comment:
I'm guessing this is the wrong place? As it's in TPMT_PUBLIC

Reply:
Weird, I thought I dropped this hunk.. I noticed it as well.